src/lib/nexus-schema-stateful/stateful-nexus-schema.ts

```diff
@@ -112,6 +115,7 @@ export function createNexusSchemaStateful() {
   ): NexusSchema.core.NexusScalarTypeDef<TypeName> {
     const typeDef = NexusSchema.scalarType(config)
     state.types.push(typeDef)
+    state.scalars[typeDef.name] = new GraphQL.GraphQLScalarType(config)
```

Comment:
Why are we creating a new GraphQL scalar type instance here?

Reply:
Because we don't want plugin authors to have to deal with two types of scalars: Nexus and GraphQL.js.